### DISCUSSION TITLE
CPU 不支持AVX instruction set问题咨询

### DISCUSSION THREAD

**Discussion by tianhao1026** [apache/doris#18555] [A - General / Q&A]
The CPU does not support AVX instruction set.
For example, if your CPU does not support AVX2, you need to rebuild the Doris BE with: USE_AVX2=0 sh build.sh --be
请问这种情况影响使用吗？还是说不影响使用但影响性能

**Comment by LemonLiTree** [comment]
It has no effect on use, but the difference in performance has not been tested yet

**Comment by qiaohanqing** [comment]
请问这个问题解决了吗？
